package.json

```diff
@@ -1,83 +1,85 @@
+    "test:src": "tsdx test",
+    "test:client": "npm_scripts/test-client.sh",
     "tdd": "npm run test -- --watch",
+    "lint": "tsdx lint",
+    "format": "npm run lint -- --fix",
+    "prepare": "husky install && tsdx build",
+    "size": "size-limit",
+    "analyze": "size-limit --why",
     "check-coverage": "cross-env CI=true NODE_ENV=test jest --coverage",
     "publish-coverage": "codecov",
     "commit": "git-cz",
+    "commit:lint-message": "commitlint --edit \"$1\"",
     "semantic-release": "semantic-release",
-    "travis-deploy-once": "travis-deploy-once"
+    "release-candidate": "npm_scripts/release-candidate.sh"
   },
-  "homepage": "https://github.com/skedify/pci-dss-sanitizer",
-  "author": {
-    "name": "Skedify",
-    "email": "info@skedify.co",
-    "url": "http://skedify.co"
+  "prettier": {
+    "printWidth": 80,
+    "semi": true,
+    "singleQuote": true,
+    "trailingComma": "es5"
   },
+  "homepage": "https://github.com/skedify/pci-dss-sanitizer",
   "repository": {
     "type": "git",
     "url": "https://github.com/skedify/pci-dss-sanitizer.git"
   },
-  "bugs": {
-    "url": "https://github.com/skedify/pci-dss-sanitizer/issues"
+  "author": {
+    "name": "Skedify",
+    "email": "info@skedify.co",
+    "url": "https://skedify.co"
   },
+  "module": "dist/pci-dss-sanitizer-new.esm.js",
+  "size-limit": [
+    {
+      "path": "dist/pci-dss-sanitizer-new.cjs.production.min.js",
+      "limit": "10 KB"
+    },
+    {
+      "path": "dist/pci-dss-sanitizer-new.esm.js",
+      "limit": "10 KB"
+    }
+  ],
   "keywords": [
     "pci-dss",
     "sanitize"
   ],
-  "files": [
-    "lib"
-  ],
-  "license": "MIT",
   "dependencies": {
     "luhn": "^2.4.1",
     "multipipe": "^4.0.0"
   },
   "devDependencies": {
-    "@commitlint/cli": "^7.2.0",
-    "@commitlint/config-conventional": "^7.1.2",
-    "babel": "^6.23.0",
-    "babel-eslint": "^7.1.1",
-    "babel-plugin-syntax-trailing-function-commas": "^6.22.0",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.23.0",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "codecov": "^3.1.0",
-    "commitizen": "^3.1.1",
-    "cross-env": "^5.2.0",
-    "cz-conventional-changelog": "^2.1.0",
-    "eslint": "^5.6.1",
-    "eslint-config-prettier": "^3.1.0",
-    "eslint-plugin-better": "^0.1.5",
-    "eslint-plugin-filenames": "^1.3.2",
-    "eslint-plugin-prettier": "^3.0.0",
-    "eslint-plugin-require-in-package": "^1.0.3",
-    "husky": "^1.1.1",
-    "jest": "^23.6.0",
-    "prettier": "^1.14.3",
-    "rimraf": "^2.6.2",
-    "rollup": "^0.41.4",
-    "rollup-plugin-babel": "^2.7.1",
-    "rollup-plugin-eslint": "^5.0.0",
-    "rollup-plugin-node-resolve": "^2.0.0",
-    "rollup-watch": "^3.2.2",
-    "semantic-release": "^15.13.16",
-    "travis-deploy-once": "^5.0.9"
+    "@commitlint/cli": "^12.1.4",
+    "@commitlint/config-conventional": "^12.1.4",
+    "@rollup/plugin-node-resolve": "^9.0.0",
```

Comment:
This can be upgraded (major version to 13 if that works ofc.)

Reply:
Not sure if related to this dependency, but after installing, I'm receiving this warning:

![image](https://user-images.githubusercontent.com/5346497/121142774-a06ca600-c83c-11eb-8ae6-16ea6039d711.png)

Is this something we can change, or is that part of the tsdx config itself?

Reply:
That is part of tsdx, but maybe it's possible to configure ourselves as well

Reply:
Updated, see https://github.com/formium/tsdx/issues/981

Reply:
Also, dropped `plugin-node-resolve` since we didn't really need it apparently.